### PR TITLE
[docs/enhancmente/fixing minor typos] - kubernetes-client/python - fi…

### PR DESCRIPTION
**Pull request by tmm88**
#### [docs/enhancmente/fixing minor typos] - kubernetes-client/python - fixing small and minor typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,13 +124,13 @@ Key:
...
+Kubernetes supports [three minor releases](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) at a time. "Support" means we expect users to be running that version in production, though we may not port fixes back before the latest minor version. For example, when v1.3 comes out, v1.0 will no longer be supported. In consistent with the Kubernetes support policy, we expect to support **three GA major releases** (corresponding to three Kubernetes minor releases) at a time.
 
-Note: There would be no maintenance for alpha/beta releases except the latest one.
+Note: There would be no maintenance for alpha/beta releases except for the latest one.
```
Comment (micw523) [inline README.md]: Except and except for are often considered the same given that the phrase is followed by a noun. See [the grammar post here](https://dictionary.cambridge.org/us/grammar/british-grammar/except-or-except-for). In this case, the correct meaning is "there would be no maintenance in addition to the last one", so the word "besides" is more accurate.

File: README.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ Participation in the Kubernetes community is governed by the [CNCF Code of Condu
 ### SSLError on macOS
 
 If you get an SSLError, you likely need to update your version of python. The
-version that ships with macOS may not be supported.
+the version that ships with macOS may not be supported.
```
Comment (micw523) [inline README.md]: Repetitive "the" is not needed.

File: README.md [hunk 3]
```diff
@@ -159,10 +159,10 @@ You'll need a version with OpenSSL version 1.0.0 or later.
 
 ### Hostname doesn't match
 
-If you get an `ssl.CertificateError` complaining about hostname match, your installed packages does not meet version [requirements](requirements.txt).
-Specifically check `ipaddress` and `urllib3` package versions to make sure they met requirements in [requirements.txt](requirements.txt) file.
+If you get an `ssl.CertificateError` complaining about hostname match, your installed packages do not meet version [requirements](requirements.txt).
+Specifically, check `ipaddress` and `urllib3` package versions to make sure they met requirements in [requirements.txt](requirements.txt) file.
```
Comment (micw523) [inline README.md]: met -> meet

File: README.md [hunk 4]
```diff
@@ -124,13 +124,13 @@ Key:
...
 
-Note: There would be no maintenance for alpha/beta releases except the latest one.
+Note: There would be no maintenance for alpha/beta releases except for the latest one.
 
 ## Community, Support, Discussion
 
-If you have any problem on using the package or any suggestions, please start with reaching the [Kubernetes clients slack channel](https://kubernetes.slack.com/messages/C76GB48RK/), or filing an [issue](https://github.com/kubernetes-client/python/issues) to let us know. You can also reach the maintainers of this project at [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery), where this project falls under.
+If you have any problem with using the package or any suggestions, please start with reaching the [Kubernetes clients slack channel](https://kubernetes.slack.com/messages/C76GB48RK/), or filing an [issue](https://github.com/kubernetes-client/python/issues) to let us know. You can also reach the maintainers of this project at [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery), where this project falls under.
```
Comment (micw523) [inline README.md]: I'd rather phrase it as "If you encounter any problem using the package, or have any suggestions"

File: README.md (hunk 1 above)
Comment (tmm88) [reply README.md]: revision done. waiting for further feedback.

File: README.md (hunk 2 above)
Comment (tmm88) [reply README.md]: revision done. waiting for further feedback.

File: README.md (hunk 3 above)
Comment (tmm88) [reply README.md]: revision done. waiting for further feedback.

File: README.md (hunk 4 above)
Comment (tmm88) [reply README.md]: revision done. waiting for further feedback.

**Comment by tmm88** [comment]
thanks for the insightful feedback. looking forward

File: README.md [hunk 5]
```diff
@@ -94,7 +94,7 @@ supported versions of Kubernetes clusters.
 
 Key:
 
-* `✓` Exactly the same features / API objects in both client-python and the Kubernetes
+* `✓`The same features / API objects in both client-python and the Kubernetes
```
Comment (micw523) [inline README.md]: I'd rather keep the space

File: README.md [hunk 6]
```diff
@@ -124,13 +124,13 @@ Key:
...
+Kubernetes supports [three minor releases](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) at a time. "Support" means we expect users to be running that version in production, though we may not port fixes back before the latest minor version. For example, when v1.3 comes out, v1.0 will no longer be supported. In consistent with the Kubernetes support policy, we expect to support **three GA major releases** (corresponding to three Kubernetes minor releases) at a time.
 
-Note: There would be no maintenance for alpha/beta releases except the latest one.
+Note: There would be no maintenance in addition to the last one.
```
Comment (micw523) [inline README.md]: We still maintain previous two minor stable versions - so we should say "there would be no maintenance for alpha/beta releases besides/in addition to the latest one"

File: README.md [hunk 7]
```diff
@@ -124,13 +124,13 @@ Key:
...
 
 ## Community, Support, Discussion
 
-If you have any problem on using the package or any suggestions, please start with reaching the [Kubernetes clients slack channel](https://kubernetes.slack.com/messages/C76GB48RK/), or filing an [issue](https://github.com/kubernetes-client/python/issues) to let us know. You can also reach the maintainers of this project at [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery), where this project falls under.
+If you have any problem any problem on the package or have any, please start with reaching the [Kubernetes clients slack channel](https://kubernetes.slack.com/messages/C76GB48RK/), or filing an [issue](https://github.com/kubernetes-client/python/issues) to let us know. You can also reach the maintainers of this project at [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery), where this project falls under.
```
Comment (micw523) [inline README.md]: Um... proofreading is important?

File: README.md (hunk 3 above)
Comment (micw523) [reply README.md]: it's in make sure they met requirements -> make sure they meet requirements

**Comment by roycaihw** [comment]
Closing due to inactivity. Please reopen when you sign the CLA

/close

**Pull request closed without merging**